src/core/request.ts

```diff
@@ -12,10 +17,13 @@ type ConfigNonFullResponse = AxiosRequestConfig & { fullResponse?: false };
 export class Request {
   request: AxiosInstance;
 
-  constructor() {
+  constructor();
+  constructor(proxy: AxiosProxyConfig);
+  constructor(proxy?: AxiosProxyConfig) {
     this.request = axios.create({
       paramsSerializer,
       withCredentials: true,
+      proxy
```

Comment:
Please don't pass it if it's undefined, it causes tests to fail.
You can do something like:
```javascript
this.request = axios.create({
   paramsSerializer,
   withCredentials: true,
   ...(proxy && { proxy })
});
```